CSharpSourceCode/Battle/TOWBattleUtilities.cs

```diff
@@ -34,7 +32,24 @@ public static void DamageAgentsInArea(Vec2 center, float radius, int minDamage,
                 }
             }
         }
-
```

Comment:
@hunharibo 

Is there a way to replicate this in the new ability system without creating separate logic?